### PR TITLE
fix(push,cli): fixed push & gitflow command

### PR DESCRIPTION
**Pull request by leorolland**
- Removed alias
- added /usr/bin/gitflow script
- fixed push command
- reimplemented gitlab.py instanciation strategy

### CODE REVIEW

File: gitflow_toolbox/common/gitlab.py [hunk 1]
```diff
@@ -41,9 +37,11 @@ def project(self) -> Project:
 class RemoteGitlab(gitlab.Gitlab, metaclass=Singleton):
 
     __project = None
+    private_token = None
 
     def __init__(self):
-        super().__init__(get_env("REMOTE_GITLAB_URL"), get_env("REMOTE_GITLAB_PRIVATE_TOKEN"))
+        self.private_token = get_env("REMOTE_GITLAB_PRIVATE_TOKEN", "GITLAB_PRIVATE_TOKEN", "CI_JOB_TOKEN")
```
Comment (ryshu) [inline gitflow_toolbox/common/gitlab.py]:
```suggestion
        self.__private_token = get_env("REMOTE_GITLAB_PRIVATE_TOKEN", "GITLAB_PRIVATE_TOKEN", "CI_JOB_TOKEN")
```

File: gitflow_toolbox/common/gitlab.py [hunk 2]
```diff
@@ -10,15 +10,11 @@
...
 
     def __init__(self):
-        # If running in a gitlab CI job
-        if os.getenv("GITLAB_CI"):
-            # Use provided token, if not available, use default token from job
-            private_token = get_env("GITLAB_PRIVATE_TOKEN", "CI_JOB_TOKEN")
-            super().__init__(get_env("CI_SERVER_URL"), private_token)
-        else:
-            super().__init__(get_env("GITLAB_URL"), get_env("GITLAB_PRIVATE_TOKEN"))
+        self.private_token = get_env("GITLAB_PRIVATE_TOKEN", "CI_JOB_TOKEN")
```
Comment (ryshu) [inline gitflow_toolbox/common/gitlab.py]:
```suggestion
        self.__private_token = get_env("GITLAB_PRIVATE_TOKEN", "CI_JOB_TOKEN")
```

**Review by ryshu — CHANGES_REQUESTED** [review]

File: gitflow_toolbox/common/gitlab.py (hunk 2 above)
Comment (leorolland) [reply gitflow_toolbox/common/gitlab.py]: `private_token` attribute is used to generate a pre-authenticated url for `push` (push.py) and so needs to be accessible

Comment (leorolland) [reply gitflow_toolbox/common/gitlab.py]: Maybe we can refactor the authenticated URL generation into a class method :)

Comment (ryshu) [reply gitflow_toolbox/common/gitlab.py]: No, it's ok for me

**Pull request merged**